pkg/validators/readiness/readiness.go

```diff
@@ -36,6 +39,10 @@ type ReadinessValidator struct {
 	ignoreMissingResources bool
 }
 
+func (v *ReadinessValidator) GetName() string {
+	return ValidatorName
+}
+
 // validates all the resources from readinesslist are ready
 func (v *ReadinessValidator) Validate(ctx context.Context, resources []unstructured.Unstructured) (violations []common.Violation, err error) {
```

Comment:
Please remove the name from the return variables

Reply:
I'm not sure what you mean